docs/_extra/api-reference/schemas/group.yaml

```diff
@@ -35,10 +35,30 @@ Group:
       description: Indicates whether a group's annotations are world-readable
     groupid:
       type: string
-      description: <mark>NEW/EXPERIMENTAL</mark> Authority-unique identifier that may be set for groups that are owned by a third-party authority. This field is currently present but unused for first-party-authority groups.
+      description: Authority-unique identifier that may be set for groups that are owned by a third-party authority. This field is currently present but unused for first-party-authority groups.
     scoped:
       type: boolean
       description: Whether or not this group has URL restrictions for documents that may be annotated within it. Non-scoped groups allow annotation to documents at any URL
+    scopes:
+      description: >
+        **EXPANDABLE** Information about the URL restrictions for annotations within this group. Only present if expanded
+      type: object
+      required:
+        - enforced
+        - uri_patterns
+      properties:
+        enforced:
+          type: boolean
+          description: >
+            When `true`, incoming annotations for this group are only allowed for documents
+            whose URIs match one of the `uri_patterns` defined for this group. If
```

Comment:
Documents have a main URI as well as associated URIs gleaned from the page but only the `target_uri` (or equivalently, `uri`) properties have to match the patterns. Perhaps it would be better to say "incoming annotations for this group are only allowed if the `uri`/`target_uri` properties matches ..."

Reply:
Thank you! This is very useful. I shall change.

Reply:
With a fresher brain this morning, I rewrote this section. What think you?:

![image](https://user-images.githubusercontent.com/439947/52129392-66a08a80-2605-11e9-81fd-70c0e09dd19f.png)

Reply:
Much better!